### ISSUE TITLE
Screen reader issues

### ISSUE THREAD

**Issue by devinprater** [zwbetz-gh/cupper-hugo-theme#34]
**Describe the bug**

Using a screen reader, one cannot tell the difference between submitted date and updated date on articles, and headings that are in the table of contents have a link after them that just goes back to that same heading, effectively going nowhere.

**To Reproduce**

1. Go to [https://devinprater.github.io/2020-04-16-writing-richly/](This page on my site).
2. Using a screen reader, notice that "Apr 16, 2020 " and "Jun 9, 2020" have no other text defining them as the submit and update date.
3. Scroll down to the "What’s this rich text and who needs all that?" heading. After that section, you'll find a link called "This What’s this rich text and who needs all that? section". That is redundant, and the link just puts focus back on that heading.

**Expected behavior**

I expect the "published" and "updated" text to set apart the first and second date, and there to be no link after headings that don't practically do anything.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] MacOS 10.15
 - Browser [e.g. chrome, safari] Safari
 - Version [e.g. 22] Version of what? Browser? Version 13.1.1 (15609.2.9.1.2)

**Smartphone (please complete the following information):**

**Additional context**

Thanks so much for this theme. There are only a few that even focus on accessibility, and I hope this theme is even made for blogging in the first place, as there seem to be a few that are for other purposes.

**Comment by zwbetz-gh** [comment]
Thanks for bringing this up. It has been fixed in https://github.com/zwbetz-gh/cupper-hugo-theme/commit/5e2ff37e94d1229ab801a7f2ef3aef3513726127.

**Comment by zwbetz-gh** [comment]
By the way, I'm digging your blog.

**Comment by devinprater** [comment]
Thanks so much! Now if I can get tags to work, my transfer from Jekyll to Hugo will be pretty complete. The tags page:

https://devinprater.github.io/tags/ <https://devinprater.github.io/tags/>

Doesn’t show any tags or pages of those tags. If it does show them visually, that may be another accessibility issue.

**Comment by devinprater** [comment]
And now the tags works… somehow. Yay! Now I can start working on more content! The only problem left is the link after the heading. It went from the heading name to saying, for example, on https://devinprater.github.io/2020-04-16-writing-richly/ <https://devinprater.github.io/2020-04-16-writing-richly/>

"2020-04-16-writing-richly”

After the "What’s this rich text and who needs all that?” heading.

**Comment by zwbetz-gh** [comment]
Good deal on the tags working. If that is indeed an issue, you'll need to open a separate GitHub issue here. 

What you may be seeing on the heading links is a _helper_ link that let's you quickly get the url fragment from that point in the page. 

I can look into making that configurable, then you could turn it off in your case.

**Comment by devinprater** [comment]
Yeah, the tags are fixed. The “helper link” I think should be configurable. thanks so much.

**Comment by zwbetz-gh** [comment]
Hiding header links is now configurable. See https://github.com/zwbetz-gh/cupper-hugo-theme/commit/e9a24f95ec8e51a3e1ac643249ec1333321772d2

Pull the latest theme changes, then add this to your config:

```yml
params:
  hideHeaderLinks: true
```

**Comment by devinprater** [comment]
Thanks so much, it works!

**Comment by zwbetz-gh** [comment]
Good deal 👍
